packages/atomic/src/locales.json

```diff
@@ -131,6 +131,14 @@
     "zh-cn": "显示较少的值",
     "zh-tw": "顯示較少的值"
   },
+  "showLessFacetResults": {
+    "en": "Show fewer results for the {{label}} facet",
```

Comment:
As far as terminology, should we use facet results or facet values? e.g. `Show fewer values ...`

Reply:
Good point,  that was from the JSUI, I'll change it in the upcoming PR